### PR TITLE
Dev Table

### PR DESCRIPTION
**Pull request by olavtar**
Checking the provider name and namespace to show only bound apps based on the provider name.
Also did some reformatting based on the ESLint suggestions. 

Signed-off-by: Olga Lavtar <olavtar@redhat.com>

### CODE REVIEW

File: src/components/instanceListPage.jsx [hunk 1]
```diff
@@ -18,29 +18,28 @@ import {
...
 import {
   crunchyProviderName,
   crunchyProviderType,
   cockroachdbProviderType,
   mongoProviderName,
   mongoProviderType,
   cockroachdbProviderName,
-} from '../const'
-import { DBAAS_PROVIDER_KIND } from '../catalog/const'
+} from '../const.ts'
```
Comment (yzhao583) [inline src/components/instanceListPage.jsx:29]: @olavtar Can you run `yarn format` and commit the change?

File: src/components/instanceListPage.jsx [hunk 2]
```diff
@@ -288,9 +280,9 @@ const InstanceListPage = () => {
             </Title>
           </EmptyState>
         ) : (
-          <React.Fragment>
+          <>
             {noInstances ? (
-              <React.Fragment>
```
Comment (yzhao583) [inline src/components/instanceListPage.jsx:293]: @olavtar Why the React.Fragment is removed here?

**Review by yzhao583 — CHANGES_REQUESTED** [review]
@olavtar Could you run `yarn format` and commit the change? Thanks

File: src/components/instanceListPage.jsx (hunk 2 above)
Comment (olavtar) [reply src/components/instanceListPage.jsx:293]: There is a new, shorter syntax you can use for declaring fragments. It looks like empty tags.
[](https://reactjs.org/docs/fragments.html)

File: src/components/instanceListPage.jsx (hunk 1 above)
Comment (yzhao583) [reply src/components/instanceListPage.jsx:29]: @olavtar Can you use the "eslintrc" as the config file for eslint

**Pull request merged**